lib/go/test/tests/client_error_test.go

```diff
@@ -411,7 +411,7 @@ func TestClientReportTTransportErrors(t *testing.T) {
 		if !prepareClientCallReply(protocol, i, err) {
 			return
 		}
-		client := errortest.NewErrorTestClientProtocol(transport, protocol, protocol)
+		client := errortest.NewErrorTestClient(thrift.NewTStandardClient(protocol, protocol))
```

Comment:
Does this means the change is still not backwards compatible with previously written customer code?

Reply:
No, it's backwards compatible now (see [here](https://github.com/dcelasun/thrift/blob/555efe5aefe9619a900471e56e86906d40bc96b9/compiler/cpp/src/thrift/generate/t_go_generator.cc#L1889-L1930)), the test simply uses the new method.

I've also tested it with the integration of tests of a real, production app. It works as expected.

Reply:
Not having to change the test is a better proof that the original method works :)

Reply:
Good point :) I'll add both, but it'll have to wait till tomorrow as I've already left the office.